aiocouch/remote.py

```diff
@@ -256,6 +267,65 @@ def endpoint(self):
         )
 
 
+class RemoteAttachment(object):
+    def __init__(self, document, id):
+        self._document = document
+        self.id = id
+        self.content_type = None
+
+    @property
+    def endpoint(self):
+        return f"{self._document.endpoint}/{_quote_id(self.id)}"
+
+    @raises(401, "Read privilege required for document '{document_id}'")
+    @raises(403, "Read privilege required for document '{document_id}'")
+    async def _exists(self):
+        try:
+            resp, data = await self._document._database._remote._head(
+                self.endpoint, return_response=True
+            )
+            return resp.headers["Content-Type"], True
```

Comment:
Because initially, `content_type` was a property of `Attachment` only (not `RemoteAttachment`). It got mixed in the last version I pushed.

Since it's in `RemoteAttachment` now, I'll take your proposal :+1: